apps/web/src/styles/globals.css

```
@@ -43,56 +43,146 @@
   --animate-slide-up: slideUp 0.3s ease-out;
   --animate-pulse-subtle: pulseSubtle 3s ease-in-out infinite;
 
+  /* Side-to-side scroll for overflow text. Translates the inner span between
+     0 and -(scrollWidth - clientWidth), which is set as a CSS custom property
+     `--marquee-shift` by the consumer. Pauses briefly at each end so users
+     can read both edges. Activated only when the consumer adds the
+     animate-marquee utility (typically on :hover). */
+  --animate-marquee: marquee 9s ease-in-out infinite;
+
   @keyframes fadeIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)